packages/eas-cli/src/commands/branch/delete.ts

```diff
@@ -115,7 +115,7 @@ export default class BranchDelete extends EasCommand {
       ({ name } = await promptAsync({
         type: 'text',
         name: 'name',
-        message: 'Please enter the name of the branch to delete:',
+        message: 'Enter the name of the branch to delete:',
```

Comment:
Can we change these, and the one(s) below, to 'Provide' as well for consistency. Compare:

'Provide the name of the channel to delete:'